src/clarity.rs

```diff
@@ -58,8 +58,22 @@ use burnchains::Txid;
 use chainstate::stacks::boot::{STACKS_BOOT_CODE_MAINNET, STACKS_BOOT_CODE_TESTNET};
 use util::boot::{boot_code_addr, boot_code_id};
 
-use core::BLOCK_LIMIT_MAINNET;
-use core::HELIUM_BLOCK_LIMIT;
+const BLOCK_LIMIT_MAINNET_STACKS20: ExecutionCost = ExecutionCost {
```

Comment:
Style nit: `BLOCK_LIMIT_MAINNET_2_0`.

Reply:
Also, why is this here, and not in `src/core/mod.rs`?

Reply:
All these cases just use the `core` definition as of 340985579